vm/systemSmartContracts/auction.go

```diff
@@ -692,10 +738,7 @@ func (s *stakingAuctionSC) stake(args *vmcommon.ContractCallInput) vmcommon.Retu
 				continue
 			}
 
-			maxStakePerNode, ok := big.NewInt(0).SetString(string(args.Arguments[i]), conversionBase)
-			if !ok {
-				continue
-			}
+			maxStakePerNode := big.NewInt(0).SetBytes(args.Arguments[i])
```

Comment:
it is pretty hard to make the arguments and set the big int as bytes - it is much easier for the user to send this value as string in the txData